### PR TITLE
added favicons, added hover on logo, linked fonts

### PR DESCRIPTION
**Pull request by philipkobernik**
I opened the PR in order to add comments

### CODE REVIEW

File: components/meta.js [hunk 1]
```diff
@@ -13,20 +13,21 @@ export default function Meta() {
         rel="icon"
         type="image/png"
         sizes="32x32"
-        href="/favicon/favicon-32x32.png"
+        href="/favicon/favicon_brico_32x32.png"
```
Comment (philipkobernik) [inline components/meta.js]: maybe just keep the name but replace the existing files?

either way, looks like there are a few more favicon files to replace in the public/favicon folder.

File: sketches/logo.js [hunk 2]
```diff
@@ -62,6 +73,11 @@ const draw = p5 => {
   p5.background(255);
 
   for (let i = 0; i < ps.length; i++) {
+    if (p5.mouseX > 0 && p5.mouseX < squareSize*bricolage.length && p5.mouseY > 0 && p5.mouseY < squareSize*bricolage.length){
+       ps[i].changeCol = true;
+     } else {
+       ps[i].changeCol = false;
+     }
```
Comment (philipkobernik) [inline sketches/logo.js]:
```
let isHovering = p5.mouseX > 0 && p5.mouseX < squareSize*bricolage.length && p5.mouseY > 0 && p5.mouseY < squareSize*bricolage.length;
ps[i].changeCol = isHovering;
```

or even bettah:
`ps[i].changeCol = isHovering(whateverArgs);`

File: components/meta.js (hunk 1 above)
Comment (sambourgault) [reply components/meta.js]: yep good idea. and yes more favicons to produce.

**Comment by philipkobernik** [comment]
once you push up commits to address any of these changes, you should should click the big green "Merge Pull Request" button and the changes will be automagically merged to master branch and deployed to bricolage.mat.ucsb.edu

File: sketches/logo.js (hunk 2 above)
Comment (sambourgault) [reply sketches/logo.js]: I added a isHovering function

**Pull request merged**